controller/controller_server/utils.py

```diff
@@ -339,12 +448,12 @@ def validate_unpublish_volume_request(request):
     logger.debug("validating unpublish volume request")
 
     logger.debug("validating volume id")
-    if len(request.volume_id.split(config.PARAMETERS_OBJECT_ID_DELIMITER)) != 2:
+    if len(request.volume_id.split(config.PARAMETERS_OBJECT_ID_DELIMITER)) not in (2, 3):
```

Comment:
(2, 3) is similar to using a magic number. I suggest wrapping it with a CAP_LETTER_VARIABLE.
Also, why using a tuple and not a set?

Reply:
done